package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "etherpad",
+  "description": "A free and open source realtime collaborative editor",
+  "homepage": "https://etherpad.org",
+  "keywords": [
+    "etherpad",
+    "realtime",
+    "collaborative",
+    "editor"
+  ],
+  "bin": {
+    "etherpad-healthcheck": "src/bin/etherpad-healthcheck",
+    "etherpad-lite": "src/bin/run"
+  },
+  "scripts": {
+    "lint": "pnpm --filter ep_etherpad-lite run lint",
+    "test": "pnpm --filter ep_etherpad-lite run test",
+    "test-container": "pnpm --filter ep_etherpad-lite run test-container",
+    "dev": "pnpm --filter ep_etherpad-lite run dev",
+    "prod": "pnpm --filter ep_etherpad-lite run prod",
+    "ts-check": "pnpm --filter ep_etherpad-lite run ts-check",
+    "ts-check:watch": "pnpm --filter ep_etherpad-lite run ts-check:watch"
+  },
+  "dependencies": {
+    "ep_etherpad-lite": "workspace:./src"
```

Comment:
it would be cleaner to rename the folder. however, as src is used quite a lot this will result in a bigger refactoring

Reply:
We could leave it as is for now but this looks very promising.